src/Core/SonarLintImageMoniker.cs

```diff
@@ -27,6 +27,11 @@ namespace SonarLint.VisualStudio.Core
     /// </summary>
     public readonly struct SonarLintImageMoniker
     {
+        /// <summary>
+        /// Based on the information inside sonarlint-visualstudio\src\IssueViz\Monikers.imagemanifest
+        /// </summary>
+        public static SonarLintImageMoniker OfficialSonarLintMoniker = new SonarLintImageMoniker(new Guid("19682eb3-4922-49e2-8f9b-e3edb43f656c"), 0);
```

Comment:
I'm not sure if this is the best place for it but it was either this or a random new constants class.

Reply:
Also will fix the SC comment in the next commit